chalk-rust-ir/src/lib.rs

```diff
@@ -130,8 +138,11 @@ pub struct TraitDatumBound<I: Interner> {
 pub struct TraitFlags {
     pub auto: bool,
     pub marker: bool,
+    /// JRL What does upstream trait mean?
     pub upstream: bool,
+    /// JRL What fundamental traits are there? Does this just mean #[fundamental] like `Box`?
```

Comment:
Currently there are none :) It would be the equivalent of the struct fundamental though